docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+    - [Security Model](#security-model)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Alternatives](#alternatives)
+  - [Upgrade Strategy](#upgrade-strategy)
+  - [Additional Details](#additional-details)
+    - [Test Plan [optional]](#test-plan-optional)
+    - [Graduation Criteria [optional]](#graduation-criteria-optional)
+      - [Alpha](#alpha)
+      - [Beta](#beta)
+      - [Stable](#stable)
+    - [Version Skew Strategy](#version-skew-strategy)
+  - [Implementation History](#implementation-history)
+
+## Glossary
+
+Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/reference/glossary.html).
+
+If this proposal adds new terms, or defines some, make the changes to the book's glossary when in PR stage.
+
+## Summary
+
+This proposal is for the support of the Windows nodes with CAPI and [infrastructure providers](https://cluster-api.sigs.k8s.io/reference/glossary.html#infrastructure-provider) that wish to support Windows [OS](https://cluster-api.sigs.k8s.io/reference/glossary.html#operating-system) worker nodes.  
+Windows has been Stable in Kubernetes since 1.14 and is supported in mix worker node scenarios.  
+
+Windows support will provide the ability to add Windows nodes to a [workload cluster](https://cluster-api.sigs.k8s.io/reference/glossary.html#workload-cluster).  Windows nodes are configured differently across 
+solutions and the centralization of the scripts and guidance in the Cluster API solution will provide 
+guidance for the community on best practices for configuring Windows nodes at scale.
+
+Windows node support has some unique challenges because of the current limitations of Windows Containers. 
+Windows containers do not have privileged containers which means that configuration and access to the host
+machine must be done at provisioning time.  
+
+An example of this is kube-proxy which is typically run as a Windows service on the host machine since it
+cannot be deployed as a Daemon set as it is on Linux.  To address this limitation the community has built
+tools such as the [CSI-Proxy](https://github.com/kubernetes-csi/csi-proxy), which is a CSI driver specific
+proxy. This proposal will address how to approach the configuration of components that are typically
+deployed as daemon sets when bootstrapping Windows nodes in CAPI.
+
+## Motivation
+The motivation of this project is to establish a set of scripts that will allow cloud providers to
+bootstrap Windows nodes in Cluster API using best practices. This will enable customers to deploy mixed-OS workloads.
+
+### Goals
+
+- Enable the creation and management of Windows worker nodes on workload clusters through CAPI and infrastructure providers
+- Provide community guidance and scripts for building base images with the use of Windows nodes
+- Re-use of the existing Cluster API Bootstrap Provider Kubeadm and other tools where appropriate
+
+### Non-Goals/Future Work
```

Comment:
Would it be good to call out that updates to kubeadm needed to for configuration are out-of-scope for this document (such as configuring the node with containerd)?

Reply:
Configuration of containerd is outside the scope of kubeadm. 

If we find a bug in Kubeadm that stops windows nodes from joining (it is in beta now and I haven't seen any major issues), I think it would need to resolved but I don't see any need to push features into kubeadm at this point and is outside the scope.  Is that what your referring to?

Reply:
Can this be closed?

Reply:
going to mark resolved, If still an issue please raise again